subsys/net/lib/lwm2m/Kconfig

```diff
@@ -370,6 +370,23 @@ config LWM2M_RW_JSON_SUPPORT
 	help
 	  Include support for writing JSON data
 
+
+config LWM2M_RW_SENML_JSON_SUPPORT
+	bool "support for SENML JSON writer[EXPERIMENTAL]"
```

Comment:
What is the justification for this being EXPERIMENTAL?

The content format follows the standard, right?

Reply:
True I will remove that.

Reply:
Experimental is justified in the sense that support for v1.1 is not fully in place. Experimental tells a user that you should really know what you're doing. But I guess it's only a matter of taste.